src/routes/app/(flows)/funder-onboarding/steps/build-list/build-list.svelte

```diff
@@ -28,6 +29,9 @@
     showListFirst={true}
     {projectUrlToAdd}
   />
+  <svelte:fragment slot="left-actions">
+    <Button icon={ArrowLeft} on:click={() => dispatch('conclude')}>Back</Button>
+  </svelte:fragment>
```

Comment:
Design changes LGTM, but not so sure about this "back" button implementation using `conclude` and `window.history.back`;

1. user could be linked to this flow from anywhere, including external sites, in which case it's weird to have an on-page back button. we'd have no idea where it leads to
2. the flow isn't `conclude`d, so i don't think dispatching the `conclude` event is the right one
3. Should probably be a separate PR to this one, since it doesn't have anything to do with truncating the project names.
4. We should probably just trigger the flow in a modal. For that, we'd just need to move the two flows with dedicated pages to the `flows` dir, then it should be pretty trivial to put them into a modal like many other flows already are.

Reply:
This code is already live on main, I must have accidentally branched this one off the back button issue branch.

I did window.history.back() because doing a goto('/') has a really gnarly page transition that I'm not sure why. While back() properly scrolls it to the place you were on the landing page. 

I was considering this a temporary solution. Sensible to make a new issue for it. I think the trouble I had was determining whether you came from landing page or entered the site at that flow. Using afterRoute() to detect had identical results.

Reply:
(resolving since handled by #684)